CMakeLists.txt

```diff
@@ -309,10 +309,10 @@ if(EMSCRIPTEN)
     xeus_wasm_compile_options(xr)
     xeus_wasm_link_options(xr "web,worker")
     target_link_options(xr
-        PUBLIC "SHELL: --pre-js ../src/env_vars.js"
+        PUBLIC "SHELL: --pre-js ${CMAKE_CURRENT_SOURCE_DIR}/wasm_patches/env_vars.js"
+        PUBLIC "SHELL: --post-js ${CMAKE_CURRENT_SOURCE_DIR}/wasm_patches/post.js"
         PUBLIC "SHELL: -s ERROR_ON_UNDEFINED_SYMBOLS=1"
```

Comment:
Just curious here, as I think ERROR_ON_UNDEFINED_SYMBOLS is set to `true`/`1` by default ?

https://emscripten.org/docs/tools_reference/settings_reference.html#error-on-undefined-symbols